mackerel-plugin-fluentd/lib/fluentd.go

```diff
@@ -34,6 +33,14 @@ type FluentdMetrics struct {
 	plugins []FluentdPluginMetrics
 }
 
+// MetricKeyPrefix interface for PluginWithPrefix
+func (f FluentdPlugin) MetricKeyPrefix() string {
+	if f.Prefix == "" {
+		f.Prefix = "fluentd"
+	}
+	return f.Prefix
+}
+
```

Comment:
What effects this function have?

Reply:
Thanks for your review.

This was added because it is necessary to satisfy the interface of PluginWithPrefix.
FetchMetrics and GraphDefinition already existed.
ref. https://mackerel.io/docs/entry/advanced/go-mackerel-plugin

```go
type PluginWithPrefix interface {
    FetchMetrics() (map[string]float64, error)
    GraphDefinition() map[string]mp.Graphs
    MetricKeyPrefix() string
}
```